resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+restartfile=/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/restartcounter.txt
+cur=$(head -1 $restartfile)
+initial=$cur
+timeout="300"
+source /etc/sysconfig/puppetserver
```

Comment:
This will already be sourced by the time the start command is running. start gets executed by a cli wrapper that lives here: https://github.com/janelu2/ezbake/blob/c5dd17967249f96efa81ff05a6415fa39ffb078f/resources/puppetlabs/lein-ezbake/staging-templates/cli-app.erb#L4-L17